src/components/CommentForm.jsx

```diff
@@ -11,17 +11,13 @@ class CommentForm extends Component {
   }
 
   handleChange(event) {
-    this.setState({
-      body: event.target.value
-    });
+    this.setState({ body: event.target.value });
   }
 
   handleSubmit(event) {
     event.preventDefault();
     postComment(this.state.body, this.props.discussionId);
-    this.setState({
-      body: ''
-    });
+    this.setState({ body: '' });
```

Comment:
I was just thinking about this...I am thinking about trying to reroute after posting a discussion so that the user knows it was posted.